dunwoody_disaster/views/fightScreen.py

```diff
@@ -311,6 +312,22 @@ def AddToQueue(self, action):
             self.defend_Btn.setEnabled(False)
             self.fight_Btn.setEnabled(True)
 
+    def UpdateMeters(self, player: Character, meters: dict):
+        maxHealth = player.maxHealth
+        curHealth = player.curHealth
+        healthPercentage = (curHealth / maxHealth) * 100
+        meters["health"].setPercentage(healthPercentage)
+
+        maxMagic = player.maxMagic
+        curMagic = player.curMagic
+        magicPercentage = (curMagic / maxMagic) * 100
```

Comment:
Is it possible to have a class that has no magic? This could throw a division by 0 error.

Fyi, make sure you pull this branch before making any changes.

Reply:
Yes, I ran into that issue which is why I changed the blank character's stats, but didn't believe it was worth messing with considering the scope of the project. I'll add a check to fix that.